### ISSUE TITLE
Get Snapshot

### ISSUE THREAD

**Issue by hostmit** [fritz-smh/yi-hack#72]
Hello!

Any chance to request/pull a snapshot via shell or any other way? It would be perfect to get a current photo by http://camera.ip/snapshot.jpg.

**Comment by tampiss** [comment]
That would be great.

**Comment by goprojojo** [comment]
Yes, it would be great if this feature can be included. I have a doorbell and need a screenshot when the doorbell rings.

**Comment by DeanNotDin** [comment]
I would love that as well!
I’m using the camera for monitoring a 3D printer and snapshots can be sent via telegram bot.
